config/database.yml

```diff
@@ -57,7 +57,16 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: postgres
+  database: blogdb_test
+
+  username: postgres
+
+  # The password associated with the postgres role (username).
+  password: test
+
+  host: localhost
+
+  port: 5433
```

Comment:
### OPTIONAL
- [ ] It is always good practice not to expose your database login information. Consider using the env file for all sensitive information about your application.